samples/sensor/grow_r502a/sample.yaml

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2021 Linumiz.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+sample:
+  name: grow_r502a fingerprint sensor sample
+tests:
+  sample.sensor.grow_r502a:
+    tags: sensors
+    harness: sensor
+    build_only: true
+    depends_on: serial arduino_serial gpio
```

Comment:
why does this depend on `arduino_serial`?

Reply:
I will remove it, as it already depends on serial.